### PR TITLE
US137469 - Remove `rich-text-enabled` attribute (Lit)

### PR DESCRIPTION
**Pull request by devpow112**
Remove the `rich-text-attribute` and all uses of it. The config that controlled this no longer exists. We should always show rich text HTML editors. This sadly skips the editor vdiffs for now as there is some rendering issue with the HTML that only happens in vdiff. Will create story to look into later.

Rubrics change (Polymer): https://github.com/Brightspace/d2l-rubric/pull/1821
Activities change: https://github.com/BrightspaceHypermediaComponents/activities/pull/2533
LMS change: https://github.com/Brightspace/lms/pull/21244

https://rally1.rallydev.com/#/57732444928d/custom/373260458992?detail=%2Fuserstory%2F630472779497

### CODE REVIEW

File: wds.config.js [hunk 1]
```diff
@@ -15,17 +15,49 @@ export const devServerConfig = {
 					}
 				}
 			}
-		})
+		}),
+		{
+			name: 'setup-html-editor-context',
```
Comment (devpow112) [inline wds.config.js:20]: Adds needed HTML editor context to all demo and test pages. This does not include vdiff sadly.

File: demo/rubric-editor.html [hunk 2]
```diff
@@ -31,17 +31,6 @@ <h2>Default</h2>
 			</template>
 		</d2l-demo-snippet>
 
-		<h2>Default (Rich Text Enabled)</h2>
```
Comment (devpow112) [inline demo/rubric-editor.html:34]: Not needed since it was just showing the rich text enabled state.

File: src/components/rubric-feedback.js [hunk 3]
```diff
@@ -480,50 +473,20 @@ export class RubricFeedback extends composeMixins(
 		}
 
 		let newEditor = null;
```
Comment (MitchelPaulin) [inline src/components/rubric-feedback.js:475]:
```suggestion
		let editor = null;
```

Comment (devpow112) [reply src/components/rubric-feedback.js:475]: I'll have to also change the below code
```js
		newEditor = html`<d2l-htmleditor
			id="text-area"
```
I can do that

Comment (devpow112) [reply src/components/rubric-feedback.js]: Ended up just moving it down to where that variable was used.

**Review by MitchelPaulin — APPROVED** [review]

**Review by jesseag — APPROVED** [review]

**Pull request merged**